bottlecap/src/config/mod.rs

```diff
@@ -810,12 +812,14 @@ pub mod tests {
     }
 
     #[test]
-    fn parse_dd_version() {
+    fn parse_number_or_string_dd_version_env() {
         figment::Jail::expect_with(|jail| {
             jail.clear_env();
             jail.set_env("DD_VERSION", "123");
+            jail.set_env("DD_ENV", "123456890");
```

Comment:
add one for service too? maybe change it to `parse_number_or_string_env_vars`

Reply:
sure